skype/skype-ps/skype/Get-CsOnlineUser.md

```diff
@@ -546,6 +546,16 @@ The following attributes are no longer relevant to Teams and have been dropped f
 - Phone
 - WindowsEmailAddress
 
+The following attributes are temporarily unavailable in the output when using the "-Filter" or when using the cmdlet without any input parameter:
```

Comment:
Probably better to say - ...... when used without the "-Identity" parameter:

Reply:
Done